calc.java

```diff
@@ -1,41 +1,87 @@
 import java.util.Scanner;
 class Calc {
     public static void main(String[] args) {
-        int Fnum,Snum;
-        int operation,sum;
+        //int Fnum,Snum;
+        int operation;
         Scanner sc = new Scanner(System.in);
-        System.out.println("Enter First Number: ");
+        /*System.out.println("Enter First Number: ");
         Fnum = sc.nextInt();
         System.out.println("Enter Second Number: ");
-        Snum = sc.nextInt();
-        System.out.println("Following operations can be performed" + ":\n1.Sum \n2.Subtrction \n3.Multiplication \n4.division \n5.Remainder");
+        Snum = sc.nextInt();*/
+        System.out.println("Following operations can be performed" + ":\n1.Sum \n2.Subtrction \n3.Multiplication \n4.division \n5.Remainder\n");
         System.out.print("Enter operation: ");
         operation = sc.nextInt();
         switch(operation) {
 
             case 1 :
-                System.out.println("Result = "+ (Fnum+Snum));
+                sum();
 		        break;
 		    case 2 :
-				System.out.println("Result = "+ (Fnum-Snum));
+				sub();
 		        break;
 			case 3 :
-				System.out.println("Result = "+ (Fnum*Snum));
+				mult();
 				break;
 			case 4 :
-				try {
-						System.out.println("Result = "+ (Fnum/Snum));
-				}
-				catch (ArithmeticException e) {
-					System.out.println("Divided by zero operation cannot possible");
-				}
+				div();
 				break;
 			case 5 :
-				System.out.println("Result = "+ (Fnum%Snum));
+				remainder();
 				break;
 			default:
 				System.out.println("Invalid operation");
         } 
         sc.close();
+		
+    }
+     public static void sum(){
+        int Fnum,Snum;
+        Scanner sc = new Scanner(System.in);
+        System.out.println("Enter First Number: ");
```

Comment:
code duplicity if in all cases we need to take two integers input. toh why can we take input in main. then pass them in parameters in every function based on operation.